Resources/Private/Fusion/Prototypes/Runtime/RuntimeForm.fusion

```diff
@@ -0,0 +1,29 @@
+prototype(Neos.Fusion.Form:Runtime.RuntimeForm) {
+    @class = 'Neos\\Fusion\\Form\\Runtime\\FusionObjects\\RuntimeFormImplementation'
+
+    namespace = null
+
+    data = Neos.Fusion:DataStructure
+
+    process = Neos.Fusion.Form:Runtime.SingleStepProcess
+
+    action = Neos.Fusion.Form:Runtime.ActionCollection
+
+    form = afx`
+        <Neos.Fusion.Form:Form form={form}>
+            {content}
+        </Neos.Fusion.Form:Form>
+    `
+
+    #
+    # @todo this should become a dynamic segment to allow caching of the first page
```

Comment:
Not really, since it would be possible to put data in `data` that is not cachable.

Integrators may do this if they know that the initial `data` is cacheble.

Reply:
Removed the comment. Can be a feature of a later release.